# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavcodec/avcodec.symbols
# Commit: 5147903580b8
# Full Hash: 5147903580b8f39197d67bddd5a0fb91d3670da4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-27 04:32:10
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r 0aad3393b572 -r 5147903580b8 media/ffvpx/libavcodec/avcodec.symbols
--- a/media/ffvpx/libavcodec/avcodec.symbols	Mon Feb 26 16:07:07 2024 +0000
+++ b/media/ffvpx/libavcodec/avcodec.symbols	Mon Feb 26 16:07:08 2024 +0000
@@ -30,11 +30,9 @@
 av_parser_close
 av_parser_init
 av_parser_parse2
-#ifdef MOZ_LIBAV_FFT
 av_rdft_calc
 av_rdft_end
 av_rdft_init
-#endif
 av_shrink_packet
 av_vorbis_parse_frame
 av_vorbis_parse_frame_flags